R/var_label.R

```diff
@@ -2,17 +2,26 @@
 #'
 #' @param x a vector or a data.frame
 #' @param value a character string or `NULL` to remove the label
-#'  For data frames, it could also be a named list or a character vector
-#'  of same length as the number of columns in `x`.
+#'  For data frames, with `var_labels()`, it could also be a named list or a
+#'  character vector of same length as the number of columns in `x`.
 #' @param unlist for data frames, return a named vector instead of a list
 #' @param null_action for data frames, by default `NULL` will be returned for
 #' columns with no variable label. Use `"fill"` to populate with the column name
 #' instead, or `"skip"` to remove such values from the returned list.
+#' @param recursive if `TRUE`, will apply `var_label()` on packed columns
```

Comment:
The tidyverse uses `recurse` as argument name in similar cases.

Reply:
changed